common/setup.c

```diff
@@ -55,9 +55,14 @@ bool_cmd("debug", opt_debug);
 
 io_port_t com_ports[2] = {COM1_PORT, COM2_PORT};
 
+static unsigned bsp_cpu_id = 0;
```

Comment:
`= 0` is inconsistent with the rest of the codebase

Reply:
For this variable I wanted to highlight the fact that by default the BSP CPU ID is `0`.